configs/katello/4.0.yaml

```diff
@@ -0,0 +1,100 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.0.0:
+    :redmine_version_id: 1177
+:prior_releases:
+  :3.18.0:
+    :redmine_version_id: 1272
+  :3.18.1:
+    :redmine_version_id: 1338
+  :3.18.2:
+    :redmine_version_id: 1353
+:code_name:
+:mash_scripts:
+  - katello-4.0-rhel7
+  - katello-4.0-thirdparty-candlepin-rhel7
+  - katello-4.0-el8
+  - katello-candlepin-4.0-el8
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-4.0
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/2.4
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 0.24.2
```

Comment:
Just confirmed with @chris1984 that the new hammer version will be 1.0

Reply:
No other changes to other katello repos, as far as I know.

Reply:
updated, thanks!